Geode.Angular/src/app/pages/auth/components/login/login.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+
+import { LoginCredentials } from '../../models/login.model';
+import { AuthService } from '../../auth.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'gd-login',
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.css',
+})
+export class LoginComponent {
+  credentialsForm: FormGroup = this.formBuilder.group({
+    email: ['', [Validators.required, Validators.email]],
+    password: ['', [Validators.required, Validators.minLength(6)]],
+  });
+  message: string = '';
+  isLoading: boolean = false;
+
+  constructor(
+    private authService: AuthService,
+    private formBuilder: FormBuilder
+  ) {}
+
+  loginSubmit(): void {
+    if (this.credentialsForm.valid) {
+      this.message = '';
+      this.isLoading = true;
+      this.authService
+        .login(this.credentialsForm.value as LoginCredentials)
+        .subscribe({
+          next: () => {
+            this.message = 'Successfully logged in!';
```

Comment:
This message should be provided in html depending on the variable you set here `isLoggedIn = true/false`
```suggestion
            this.isLoggedIn= true;
```

Reply:
Further navigation should be here

Reply:
I've removed it completely and now it invokes a snackbar instead (in the 'feature-auth-updates' branch)